IdentityServer/v6/docs/content/quickstarts/1_client_credentials.md

```diff
@@ -4,101 +4,125 @@ date: 2020-09-10T08:22:12+02:00
+* *HostingExtensions.cs* - configuration for ASP.NET pipeline and services.
+  Notably, the IdentityServer services are configured here and the
+  IdentityServer middleware is added to the pipeline here.
+* *Config.cs* - definitions for [resources]({{< ref "/overview/terminology#resources" >}}) and 
+[clients]({{< ref "/overview/terminology#client" >}}) used by IdentityServer
 
-You can now use your favorite text editor to edit or view the files. If you want to have Visual Studio support, you can add a solution file like this:
+You can now use your favorite text editor to edit or view the files. 
 
-```
+Add the IdentityServer project to the solution:
+```console
 cd ..
-dotnet new sln -n Quickstart
-```
-
-and let it add the IdentityServer project (keep this command in mind as we will create other projects below):
-
-```
 dotnet sln add ./src/IdentityServer/IdentityServer.csproj
 ```
 
 {{% notice note %}}
-The protocol used in this Template is *https* and the port is set to 5001 when running on Kestrel or a random one on IISExpress. You can change that in the *Properties\launchSettings.json* file. For production scenarios you should always use *https*.
-{{% /notice %}}
 
+The protocol used in this Template is *https* and the port is set to 5001 when
+running on Kestrel or a random one on IISExpress. You can change that in the
+*Properties\launchSettings.json* file. For production scenarios you should
+always use *https*. 
 
-#### Defining an API Scope
-An API is a resource in your system that you want to protect. 
-Resource definitions can be loaded in many ways, the template you used to create the project above shows how to use a "code as configuration" approach.
+{{% /notice %}}
 
-The Config.cs is already created for you. Open it, update the code to look like this::
+### Defining an API Scope
+Scope is a core feature of OAuth that allows you to express the extent or scope
+of access. Clients request scopes when they initiate the protocol, declaring
+what scope of access they want. IdentityServer then has to decide which scopes
+to include in the token. Just because the client has asked for something doesn't
+mean they should get it! There are built-in abstractions as well as
+extensibility points that you can use to make this decision. Ultimately,
+IdentityServer issues a token to the client, which then uses the token to access
+APIs. APIs can check the scopes that were included in the token to make
+authorization decisions.
+
+Scopes don't have structure imposed by the protocols - they are just
+space-separated strings. This allows for flexibility when designing the scopes
+used by a system. In this quickstart, you will create a scope that represents
+complete access to your API.
+
+Scope definitions can be loaded in many ways. This quickstart shows how to use a
+"code as configuration" approach. A minimal Config.cs was created by the
+template. Open it and add an ApiScope to the ApiScopes property:
```

Comment:
Elsewhere we have a convention for keywords -- ApiScope and ApiScopes should use italics.